src/Components/WebAssembly/Build/test/BuildIntegrationTests/BuildIntegrationTest.cs

```diff
@@ -325,6 +325,29 @@ public async Task Build_WithI8NOption_CopiesI8NAssembliesWithLinkerEnabled()
             Assert.FileDoesNotExist(result, buildOutputDirectory, "wwwroot", "_framework", "_bin", "I18N.West.dll");
         }
 
+        [Fact]
+        public async Task Build_WithCustomOutputPath_Works()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("standalone", additionalProjects: new[] { "razorclasslibrary" });
+            project.AddDirectoryBuildContent(
```

Comment:
You can write to a file named `Before.Directory.Build.props` instead of editing `Directory.Build.props`: https://github.com/dotnet/aspnetcore/pull/21128/files#diff-d162962c2f4ef3bce632f3f874e92aefR2

Reply:
I tried this at first but the values in Before.Directory.props values are initialized before things like `MSBuildThisProjectDirectory` are set so it wasn't usable for this test case.

Reply:
No, like `Before.Directory.Build.props`. It's out special thing. But what you have should be fine as well